Documentation/building/linux-instructions.md

```diff
@@ -237,3 +237,72 @@ If you want to focus on the speed optimization for high-end devices, you have to
  SET (CLR_DEFINES_DEBUG_INIT              DEBUG _DEBUG _DBG URTBLDENV_FRIENDLY=Checked BUILDENV_
 ```
 
+Additional configuring features for ARM/Linux
+=============================================
+
+Disable READYTORUN
+------------------
```

Comment:
I do not think this should be documented here like this. It will get out of date very quickly. If you have a good reason for compiling with this disabled often, make it option for the CMake scripts.